parser/parse.go

```diff
@@ -116,6 +116,11 @@ func (b *Builder) AddBuildTags(tags ...string) {
 	b.context.BuildTags = append(b.context.BuildTags, tags...)
 }
 
+func isErrPackageNotFound(err error) bool {
+	errMessage := err.Error()
+	return strings.HasPrefix(errMessage, "cannot find package") && strings.Contains(errMessage, "in any of:")
```

Comment:
is this testable in some lines of code that the strings do not change in another go release?